pkg/client/model_customer.go

```diff
@@ -28,13 +28,32 @@ type Customer struct {
 	// Customers name suffix. \"Jr\", \"PH.D.\"
 	Suffix string       `json:"suffix,omitempty"`
 	Type   CustomerType `json:"type"`
+	// Business Name for business type customers
+	BusinessName string `json:"businessName,omitempty"`
+	// Doing Business As (DBA) name for business type customers
+	DoingBusinessAs	string `json:"doingBusinessAs,omitempty"`
+	// Business Type for business type customers -
+	//	\"Individual, sole proprietor, or single-member LLC\", \"Corporation\", \"Nonprofit organization\", \"Partnership\", or \"Limited Liability Company\"
+	BusinessType string `json:"businessType,omitempty"`
+	// Employer Identification Number (EIN) for business type customers
+	Ein	string `json:"ein,omitempty"`
+	// Dun & Bradstreet D-U-N-S Number (D-U-N-S) for business type customers
+	Duns string `json:"duns,omitempty"`
```

Comment:
nit: We should uppercase the acronyms here

Reply:
This should be resolved in my latest commits